### PR TITLE
[stable/mariadb] add service annotations support

### PR DESCRIPTION
**Pull request by eddycharly**
Support for service annotations.
This is useful for things like external-dns.

### CODE REVIEW

File: stable/mariadb/values-production.yaml [hunk 1]
```diff
@@ -293,6 +293,9 @@ master:
     minAvailable: 1
     # maxUnavailable: 1
 
+  service:
```
Comment (juan131) [inline stable/mariadb/values-production.yaml:298]: Could you please add some comments explaining what's the parameter used for?

File: stable/mariadb/values-production.yaml [hunk 2]
```diff
@@ -408,6 +411,10 @@ slave:
...
```
Comment (juan131) [inline stable/mariadb/values-production.yaml:420]: Idem

**Review by juan131 — CHANGES_REQUESTED** [review]
Hi @eddycharly 

Thanks so much for your contribution!! Could you please bump the minor version in the **Chart.yaml** and address the suggestions?

Please also note that you need to sign-off your commits, otherwise the CI/CD will reject any of your contributions.

**Comment by eddycharly** [comment]
@juan131 thank you for your comments, i updated the pull request and signed the commits.

File: stable/mariadb/Chart.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mariadb
-version: 6.8.10
+version: 6.8.11
```
Comment (juan131) [inline stable/mariadb/Chart.yaml]: It seems you bumped the path version instead of the minor one:
```diff
- version: 6.8.11
+ version: 6.9.0
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks for implementing the suggestion! Just a minor change and we're ready to go ahead with the merging process

File: stable/mariadb/Chart.yaml (hunk 3 above)
Comment (eddycharly) [reply stable/mariadb/Chart.yaml]: done 👍

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks!! Approving your changes

**Review by juan131 — APPROVED** [review]

**Pull request merged**